app/_locales/en/messages.json

```
@@ -5214,6 +5214,18 @@
   "toggleEthSignOn": {
     "message": "ON (Not recommended)"
   },
+  "toggleRequestQueueDescription": {
+    "message": "When a dapp or the wallet makes a network request, it's possible that the currently selected network is not the same as what is expected. Turning this feature on will switch the network according to the origin of the request. Using switch ethereum chain will change the saved chain for the particular origin."
+  },
+  "toggleRequestQueueField": {
+    "message": "Enable or disable request queueing when network switching is required"
```

[From figma](https://www.figma.com/file/LDlI7yjKBNIY60MtQv9QS0/Amon-Hen%2FV1?type=design&node-id=6122-46694&mode=design)
```suggestion
    "message": "Select networks for each site"
```